plugins/wake/wake.plugin.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# oh-my-bash.module: wake-on-lan wrapper + autocompletion
+# wake.plugin.sh
+# Author: hoek from 0ut3r.space
+# Based on oh-my-zsh wake plugin
+
+function wake() {
+  local cfgdir="$HOME/.wakeonlan"
+  local cfgfile="$cfgdir/$1"
+
+  if [[ -z "$1" || ! -f "$cfgfile" ]]; then
+    echo "Usage: wake <device>"
+   if [[ -d "$cfgdir" ]]; then
+     echo "Available devices: $(ls "$cfgdir" | tr '\n' ' ')"
+   else
+     echo "No devices configured. Create $cfgdir directory first."
+   fi
+    return 1
+  fi
+
+  if ! command -v wakeonlan >/dev/null; then
+    echo "ERROR: 'wakeonlan' not found. Install it (https://github.com/jpoliv/wakeonlan)." >&2
+    return 1
+  fi
+
+  # Read the two fields: MAC and broadcast-IP
+  local mac bcast
+  read -r mac bcast < "$cfgfile"
```

Comment:
This can be affected by `IFS` that the user sets in the shell session. `IFS` should be explicitly specified.